runtime/org.eclipse.rcptt.ecl.data.apache.poi.impl/src/org/eclipse/rcptt/ecl/data/apache/poi/impl/internal/commands/ExcelFileService.java

```diff
@@ -77,15 +74,15 @@ public static String getCellValue(Cell cell) {
 			return "";
 		}
 		switch (cell.getCellType()) {
-		case Cell.CELL_TYPE_STRING:
+		case STRING:
```

Comment:
[minor] I remember attempting this before and it failed build as we are using an older library version.

Reply:
I downgraded POI to 3.9.0 again so this API change is not required.